src/validator_builder.py

```diff
@@ -3,6 +3,5 @@
 from src.three_by_three_validator import ThreeByThreeValidator
 
 class ValidatorBuilder():
```

Comment:
Nice job implementing the builder pattern, but I'm wondering if it's necessary at this point. I think it's good to look ahead to how you can be flexible enough to accommodate another board type, but to me it seems like overkill to do something like this preemptively, and abstracting before you have enough redundancies to drive the effort can lead to a bad abstraction. This is somewhat subjective though, so I'd be interested to hear what you and others think.

Reply:
I made the choice to start using builders once the backlog grooming showed that the client may want to have different board types and whatnot. Like you said, and I may agree with you after yesterday's workshop, this preemptive effort may just be overkill at this stage. I'm not currently convinced that it definitely is but I'm sure ill quickly be proven wrong if the client changes his mind.